week-6/Homework/mandatory/3-slideshow/task.md

```diff
@@ -0,0 +1,81 @@
+# Challenge: "Image Carousel" (React)
+
+## Challenge Overview
+
+Make a website which allows the user to navigate a set of images (first manually, then with an auto-playing slideshow).
+
+[Try this live demo!](https://cyf-image-carousel.netlify.com/)
+
+# Level 1 Challenge
+
+Make forward and back buttons to move _manually_ in that direction through a list of at least 4 images.
+
+(e.g. When the user presses forward once, the display should move ONCE to the next image.)
+
+You can use any images.
+
+You can store the images within your app or you can use links to images hosted elsewhere ("hotlinking").
+
+[Unsplash](https://unsplash.com/) is a good source of images for this challenge.
+
+Level 1 challenge screenshot example.
+
+![Example Screenshot](./example-screenshots/example-level1.png)
+
+# Level 2 Challenge
+
+Make auto-forward and auto-back buttons to _automatically_ move in that direction through the list of images.
+
+Here's a screenshot example from a completed level 2 challenge.
+
+![Example Screenshot](./example-screenshots/example-level2.png)
+
+## Try to finish the rest by yourself
+
+If you want a harder challenge, don't read the rest of this document but instead try to build the app by yourself.
+
+If you want hints, then you will find some below.
+
+## Suggested approach
+
+Here's one approach you might take to building this app.
+
+### Task: Design your layout _on paper_
+
+Design your layout on paper. Keep it very simple - this is a JavaScript challenge, not a CSS challenge.
+
+Use a layout that will be ok on a phone (but _don't_ spend time on responsive design).
+
+Keep this drawing around for reference later.
+
+### Task: Convert your layout to HTML
+
+Convert the drawing to HTML (on codepen or elsewhere) and check the buttons appear correctly.
+
+_DON'T_ add any CSS or extra markup to make it look good just now. That will only make it more difficult for you to think about your app during development.
+
+#### Task: Make the buttons work
+
+Make your buttons work to navigate forwards and backwards, manually.
+
+### Task: more buttons - automated slideshow
+
+Add the following buttons:
+
+- auto-forward
+- stop
+- auto-backwards
+
+These should allow automatic navigation through the images, say every 5 seconds.
+
+### End of basic challenge!
+
+Congratulations, you've finished the basics!
+
+- Send the URL of your hosted app to your team on Slack.
```

Comment:
Do they know how to host stuff? Maybe we can link to instructions for how to set up github pages or similar on this line, as a reminder?